app/celery/reporting_tasks.py

```diff
@@ -31,37 +32,47 @@ def create_nightly_billing(day_start=None):
         # When calling the task its a string in the format of "YYYY-MM-DD"
         day_start = datetime.strptime(day_start, "%Y-%m-%d").date()
     for i in range(0, 4):
-        process_day = day_start - timedelta(days=i)
+        process_day = (day_start - timedelta(days=i)).isoformat()
 
         create_nightly_billing_for_day.apply_async(
-            kwargs={'process_day': process_day.isoformat()},
+            kwargs={'process_day': process_day},
             queue=QueueNames.REPORTING
         )
+        current_app.logger.info(
+            f"create-nightly-billing task: create-nightly-billing-for-day task created for {process_day}"
+        )
 
 
 @notify_celery.task(name="create-nightly-billing-for-day")
 @statsd(namespace="tasks")
 def create_nightly_billing_for_day(process_day):
     process_day = datetime.strptime(process_day, "%Y-%m-%d").date()
+    current_app.logger.info(
+        f'create-nightly-billing-for-day task for {process_day}: started'
+    )
 
     start = datetime.utcnow()
     transit_data = fetch_billing_data_for_day(process_day=process_day)
     end = datetime.utcnow()
 
-    current_app.logger.info(f'create-nightly-billing-for-day {process_day} fetched in {(end - start).seconds} seconds')
+    current_app.logger.info(
+        f'create-nightly-billing-for-day task for {process_day}: data fetched in {(end - start).seconds} seconds'
+    )
 
     for data in transit_data:
         update_fact_billing(data, process_day)
 
     current_app.logger.info(
-        f"create-nightly-billing-for-day task complete. {len(transit_data)} rows updated for day: {process_day}"
+        f"create-nightly-billing-for-day task for {process_day}: "
+        f"task complete. {len(transit_data)} rows updated for {process_day}"
```

Comment:
don't think we need to keep the second `{process_day}` here

Reply:
Good spot, removed and rebased